packages/notification-center/src/store/styles/styles-provider.tsx

```diff
@@ -16,17 +15,27 @@ export const useStyles = (path: StylesPaths | StylesPaths[]): CSSInterpolation[]
     throw new Error('useStyles must be used within a StylesProvider');
   }
 
-  const getStyleByPath = (pathInStyles: StylesPaths): CSSInterpolation => {
-    const stylePart = get(stylesContext.styles, pathInStyles);
-
-    return typeof stylePart === 'function' ? stylePart({ theme, common, colorScheme }) : stylePart;
-  };
```

Comment:
moved this to the util function `getStyleByPath` it's then reused here and in the Vue package